### PR TITLE
Volvo pr Ciel day 12

### PR DESCRIPTION
**Pull request by Volvo-hackangel**

### CODE REVIEW

File: script/Ciel/Day 12/12_01_CIEL12_1_A.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ Just as I was thinking that today's knocks were a bit louder than usual,
 [sha:27beaea45ec2140eebfbc52e09898f6c2782045c]{
 -- Page 6, Offset 30880.
 -- 「おはようございます志貴さん。今朝の具合はどうですか？」
-"Good morning Shiki-san. How are you feeling this morning?"
+"Good morning, Shiki-san. How are you feeling this morning?"
```
Comment (rschlaikjer) [inline script/Ciel/Day 12/12_01_CIEL12_1_A.txt:15]: Don't love double-morning, maybe change second to `today`

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt [hunk 2]
```diff
@@ -489,7 +489,7 @@ Let's all have lunch together. I heard that the Indian pies at the café next to
 [sha:0838227d0ba55239bb3e70742d1eaf0506fc548e]{
 -- Page 127, Offset 31191.
 -- 「おっ、アーネンエルベとは通だね先輩！　あの喫茶店のマスター、どこぞのイタリア料理の達人らしいぜ！」
-"Oh, you're a connoisseur of Ahnenerbe, are you Senpai?
+"Oh, you're a connoisseur of Ahnenerbe, are you, Senpai?
```
Comment (rschlaikjer) [inline script/Ciel/Day 12/12_01_CIEL12_3B.txt:492]: Hmm only one of those latter commas can stay
> Oh, you're a connoisseur of Ahnenerbe are you, Senpai?

maybe drop that `are you` as well?
> Oh, you're a connoisseur of Ahnenerbe, Senpai?

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 12/12_01_CIEL12_1_A.txt (hunk 1 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 12/12_01_CIEL12_1_A.txt:15]: nice catch

File: script/Ciel/Day 12/12_01_CIEL12_3B.txt (hunk 2 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 12/12_01_CIEL12_3B.txt:492]: I'm going with the latter.

File: script/Ciel/Day 12/12_01_CIEL12_2B.txt [hunk 3]
```diff
@@ -374,7 +374,7 @@ I feel like I'm always the one waiting for you to come to me,
 -- Page 54, Offset 31001.
 -- 「それはそれは。嬉しいですけど、遠野くん、わたしのクラスを知らないでしょ？　３年のＢクラスですから覚えておいてくださいね」
 "Well, isn't that something?
- I'm happy to hear that, but you don't know which class I'm in, do you Tohno-kun?
+ I'm happy to hear that, but you don't know which class I'm in, do you, Tohno-kun?
```
Comment (Volvo-hackangel) [inline script/Ciel/Day 12/12_01_CIEL12_2B.txt:377]: In the context of the other one, should this comma be removed as well?

Comment (rschlaikjer) [reply script/Ciel/Day 12/12_01_CIEL12_2B.txt:377]: Saying this one aloud it seems OK to me

File: script/Ciel/Day 10/10_01_CIEL10_9.txt [hunk 4]
```diff
@@ -751,7 +751,8 @@ Are you saying I'm not contaminated?!
 [sha:902b4d50071a15718e21c3d035a0a6f7624a459f]{
 -- Page 645, Offset 28912.
 -- 「――――――！」
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_9.txt:753]: This is Noel doing the horrified gasp; not sure how to render it lmao
>Hnnngh!

maybe?

File: script/Ciel/Day 12/12_01_CIEL12_1.txt [hunk 5]
```diff
@@ -1,7 +1,8 @@
 [sha:902b4d50071a15718e21c3d035a0a6f7624a459f]{
 -- Page 1, Offset 30866.
 -- 「――――――！」
-"―――!"
+// lint-off:LintEmDashes
+"――――――!"
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_1.txt:5]: Shocked gasp; maybe 
>Whoa!

or maybe

>*Gasp*

idk

File: script/Ciel/Day 12/12_01_CIEL12_1_A.txt [hunk 6]
```diff
@@ -12,7 +12,7 @@ Just as I was thinking that today's knocks were a bit louder than usual,
 [sha:27beaea45ec2140eebfbc52e09898f6c2782045c]{
 -- Page 6, Offset 30880.
 -- 「おはようございます志貴さん。今朝の具合はどうですか？」
-"Good morning Shiki-san. How are you feeling this morning?"
+"Good morning, Shiki-san. How are you feeling this today?"
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_1_A.txt:15]: missed a word

File: script/Ciel/Day 12/12_01_CIEL12_3_1.txt [hunk 7]
```diff
@@ -126,7 +126,7 @@ I run in pursuit of my promise with Senpai.
 [sha:f44852bf871ada28c7958687838fee104d45f160]{
 -- Page 162, Offset 31286.
 -- 倒れる寸前の体に鞭をうって走った。
-I run in flight of some unknowable something that's pursing me.
+I run in flight of some unknowable something that's pursuing me.
```
Comment (Petrikow) [inline script/Ciel/Day 12/12_01_CIEL12_3_1.txt:129]: italics on something

File: script/Ciel/Day 5/05_01_CIEL05_3E.txt [hunk 8]
```diff
@@ -966,7 +966,8 @@ That school of swordsmanship...
 [sha:902b4d50071a15718e21c3d035a0a6f7624a459f]{
 -- Page 635, Offset 12531.
 -- 「――――――！」
```
Comment (Petrikow) [inline script/Ciel/Day 5/05_01_CIEL05_3E.txt:968]: shocked gasp
>Huh?!

File: script/Ciel/Day 10/10_01_CIEL10_9.txt (hunk 4 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 10/10_01_CIEL10_9.txt:753]: works for me

File: script/Ciel/Day 12/12_01_CIEL12_1.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 12/12_01_CIEL12_1.txt:5]: going with the latter

File: script/Ciel/Day 12/12_01_CIEL12_1_A.txt (hunk 6 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 12/12_01_CIEL12_1_A.txt:15]: damn im blind nice catch

File: script/Ciel/Day 12/12_01_CIEL12_1.txt (hunk 5 above)
Comment (Volvo-hackangel) [reply script/Ciel/Day 12/12_01_CIEL12_1.txt]: on second thought, I opted for something different

**Pull request merged**